src/main/java/com/amazon/dlic/auth/http/jwt/AbstractHTTPJwtAuthenticator.java

```diff
@@ -72,7 +72,7 @@ public AbstractHTTPJwtAuthenticator(Settings settings, Path configPath) {
         rolesKey = settings.get("roles_key");
         subjectKey = settings.get("subject_key");
         clockSkewToleranceSeconds = settings.getAsInt("jwt_clock_skew_tolerance_seconds", DEFAULT_CLOCK_SKEW_TOLERANCE_SECONDS);
-        requiredAudience = settings.get("required_audience");
+        requiredAudience = settings.getAsList("required_audience").toArray();
```

Comment:
wdyt about keeping this as a `List<String>` and avoid having to re-cast this back to a list in the `JwtVerifier`?

Reply:
+1. Is there any specific reason why Object array was considered for required_audience over string list?